source/developer/webhooks-incoming.rst

```diff
@@ -176,7 +176,7 @@ To send a message to a Direct Message channel, add an "@" symbol followed by the
 
 This will send a message from the account that has set up the incoming webhook to the username after the "@" symbol. For example, if you create a webhook with the user ``alice`` and send a Direct Message to ``bob`` using a webhook, it will show up as a direct message from ``alice`` to ``bob`` regardless of other settings such as username.
 
-To send a message to a different Direct Mssage channel between two other users, you can specify the channel with the user IDs for the users separated with two underscore (_) symbols. To find the user ID you can `use the ``mattermost`` command - see details `here <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-user-search>`__.
+To send a message to a different Direct Mssage channel between two other users, you can specify the channel with the user IDs for the users separated with two underscore (_) symbols. To find the user ID you can `use the ``mattermost`` command - see details `here <https://docs.mattermost.com/manage/command-line-tools.html#mattermost-user-search>`__.
```

Comment:
```suggestion
To send a message to a different Direct Message channel between two other users, you can specify the channel with the user IDs for the users separated with two underscore (_) symbols. To find the user ID you can `use ``mmctl user search``- see details `here <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-user-search>`__.
```

Reply:
We're moving to mmctl as default, so I think this URL is a better option as it avoids users landing on one page and having to go to another page. @cwarnermm, is there anything we should add to this?

Reply:
Yeah, it will be perfect. I just pointed out the deprecated link. If Mattermost is moving with a new command, that link will be perfect.

Reply:
Great catch, @justinegeffen! Yes - let's go with the mmctl command rather than the CLI one, given that CLI will be deprecated as of v6.0.

Reply:
Amazing @cwarnermm